### PR TITLE
Make site responsive (fix #1)

### LINKED ISSUE #1: Looks terrible on mobile
* Scaling is off
 * Text overflows
 * Padding is overdone
 * Navbar items completely disappear

### PR DESCRIPTION
**Pull request by AtaGowani**
# Changes
* Fixed terminal
* Added toggler
* License link fixed
* Minor organizational changes

Additions and changes made in `index.html` and `style.css`

# Screenshots

## extra sm
screen shot 2017-06-23 at 3 41 02 pm

## sm
screen shot 2017-06-23 at 3 40 49 pm

## md
screen shot 2017-06-23 at 3 40 35 pm

## lg
screen shot 2017-06-23 at 3 40 18 pm

# extra lg
screen shot 2017-06-23 at 3 40 02 pm

# Issue
See #1

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/**
+* @file css/index.css
```
Comment (karimsa) [inline css/style.css]: This doesn't match the actual filename.

File: css/style.css [hunk 2]
```diff
@@ -22,7 +28,7 @@ body {
   background: #fff;
 }
 
-.navbar-brand {
+.navbar-light .navbar-brand, .navbar-light .navbar-brand:hover, .navbar-light .navbar-brand:visited {
```
Comment (karimsa) [inline css/style.css]: CSS selectors should be on separate lines. Fix this manually for now, we'll add linting for styles soon.

File: css/style.css [hunk 3]
```diff
@@ -22,7 +28,7 @@ body {
   background: #fff;
```
Comment (karimsa) [inline css/style.css]: The background is set to white by default.

**Review by karimsa — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 3 above)
Comment (AtaGowani) [reply css/style.css]: `.bg-faded` was changing the color to `#f7f7f7`.
I'll get rid of that.

Comment (AtaGowani) [reply css/style.css]: It doesn't seem to have any side effects

File: css/style.css [hunk 4]
```diff
@@ -97,11 +111,17 @@ body {
   color: #fff;
   
   margin: 0;
-  padding: 1.5rem 3rem;
+  padding: 1.5rem 1.5rem;
```
Comment (karimsa) [inline css/style.css]: This can be reduced to `padding: 1.5rem;`

**Review by karimsa — CHANGES_REQUESTED** [review]
Can you upload screenshots of what this looks like on sm, md, and lg?

**Review by karimsa — APPROVED** [review]

**Pull request merged**